pkg/infra/kubernetes/plugin.go

```diff
@@ -42,13 +43,31 @@ func (p Kubernetes) Translate(constructGraph *core.ConstructGraph, dag *core.Res
 	// For exec units that specify their own chart, we want to render and replace
 	for dir, khChart := range klothoCharts {
 		dirToLoad := filepath.Join(p.Config.Path, dir)
-		chart, err := p.helmHelper.LoadChart(dirToLoad)
+		chartContent, err := p.helmHelper.LoadChart(dirToLoad)
+
 		if err != nil {
-			errs.Append(err)
-			continue
+			if err.Error() == "Chart.yaml file is missing" {
```

Comment:
Not a huge fan of checking error messages. Is there not a type you can `errors.Is` with?

Reply:
not sure, this was from before i took over, so i can change it